packages/web/src/scss/components/Toast/README.md

```diff
@@ -0,0 +1,247 @@
+# Toast
+
+Toast displays a brief, temporary notification that appears at a prescribed location of an application window.
+
+Toast is a composition of a few subcomponents:
```

Comment:
Is the `Toast` also a subcomponent?

Reply:
I have been waiting for this one… 🙈

Reply:
Let's keep it this way. The same formulation can be found in `FileUploader`, `Header`, `Modal`, …

Reply:
OK :)